### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [fel88/Dagre.NET#5] [General]
Based on this [discussion](https://github.com/dagrejs/dagre/issues/294), the API source is defined.
Putting the link here to reference and discuss the[ Dagre API](https://github.com/dagrejs/graphlib/wiki/API-Reference)

**Comment by GeorgeS2019** [comment]
@fel88  Is there implementation of these features in Dagre.NET? 
- [Multigraphs](https://github.com/dagrejs/graphlib/wiki/API-Reference#multigraphs)
- [CompoundGraphs](https://github.com/dagrejs/graphlib/wiki/API-Reference#compound-graphs)

How the recent cluster feature you implemented is related to these features. Thx :-)

**Reply by fel88** [reply]
> @fel88 Is there implementation of these features in Dagre.NET?

Yes, sure. These are fully implemented features (except possible hidden bugs)
